src/test/ui/backtrace.rs

```diff
@@ -6,6 +6,7 @@
 // ignore-msvc see #62897 and `backtrace-debuginfo.rs` test
 // compile-flags:-g
 // compile-flags:-Cstrip=none
+// needs-unwind
```

Comment:
This test has some abort-specific conditional compilation already, which suggests that this should work in panic="abort" environments. What was the specific error that occurred with this test?

Reply:
The error is at:
```
assert!(s.contains("stack backtrace") && s.contains(&expected("foo")),
            "bad output: {}", s);
```

Looking again, it's likely that this is because the backtrace that Fuchsia produces does not contain the symbols expected. Removing this change from this PR